### PR TITLE
Add delete forward/back word bound to ctrl delete/backspace.

### PR DESCRIPTION
**Pull request by barnybug**

### CODE REVIEW

File: lib/howl/ui/editor.moon [hunk 1]
```diff
@@ -959,7 +969,9 @@ for cmd_spec in *{
   { 'smart-tab', 'Inserts tab or shifts selected text right', 'smart_tab' }
   { 'smart-back-tab', 'Moves to previous tab stop or shifts text left', 'smart_back_tab' }
   { 'delete-back', 'Deletes one character back', 'delete_back' }
+  { 'delete-back-word', 'Deletes one word back', 'delete_back_word' }
   { 'delete-forward', 'Deletes one character forward', 'delete_forward' }
+  { 'delete-forward-word', 'Deletes one word back', 'delete_forward_word' }
```
Comment (kirbyfan64) [inline lib/howl/ui/editor.moon]: "Deletes one word **forward**"

**Comment by barnybug** [comment]
oops, fixed :-)

File: lib/howl/keymap.moon [hunk 2]
```diff
@@ -16,7 +16,9 @@
     tab:              'editor-smart-tab'
     shift_tab:        'editor-smart-back-tab'
     backspace:        'editor-delete-back'
+    ctrl_backspace:   'editor-delete-back-word'
```
Comment (shalabhc) [inline lib/howl/keymap.moon]: Could you please also add the `osx` equivalents in the section below? I think `meta_backspace` and `meta_delete` should work.

Comment (barnybug) [inline lib/howl/keymap.moon]: I think it'd be bound to Option+Backspace/Delete on the Mac - but I noticed this comment re option key in the keymap: https://github.com/howl-editor/howl/blob/master/lib/howl/keymap.moon#L152

Comment (shalabhc) [inline lib/howl/keymap.moon]: OK, Option+* variants will work too. I think Word used Command+Backspace, Atom uses Option+Backspace and Lifehacker says both work:

http://lifehacker.com/5803041/the-mac-os-x-delete-key-it-goes-both-ways

Perhaps just add commented out code using `option_backspace` etc. in the osx section then? Next time I fire up Howl on OSX I'll try and set up the the `option` key translation.

**Comment by barnybug** [comment]
Good idea. Added comment.

**Comment by nilnor** [comment]
Looks good! One nitpick, and it's the behaviour when a selection is active. Even though it probably doesn't make sense to type these shortcuts with a selection active, it would probably indicate an accidentally pressed `ctrl` key rather than anything else, in which case it would probably make sense to just delete the selection instead of attempting anything else.

**Comment by nilnor** [comment]
Oh, and disregard the failing travis for now, the memory tests have been proven to be fickle on Travis and something will have to be done, but it's unrelated to this.

**Comment by barnybug** [comment]
Thanks for the suggestion - selection behaviour improved.

**Comment by nilnor** [comment]
All right, looks good!

**Pull request merged**